piker/clearing/_paper_engine.py

```diff
@@ -32,6 +32,7 @@
 
 from .. import data
 from ..data._normalize import iterticks
+from ..data._source import uncons_fqsn
```

Comment:
uncons should likely be changed to unpack

Reply:
right, good call.